docs/aerodromes/procedural/Tamworth.md

```diff
@@ -103,8 +111,18 @@ During day VMC only, simultaneous parallel runway operations may be used between
 
 In the absense of day VMC, dependent parallel runway operations may be conducted, with an adequate separation standard in place between aircraft operating from adjacent runways.
 
+## VFR Departures/Arrivals
+VFR aircraft departing between the 160 radial through to the 295 radial should report ready **GST** or **GWT**.
```

Comment:
Might be worth explaining it from a controllers perspective? Given vatsim pilots have less experience to local procedures may be worth asking for pilots to report at these points